src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from sentry.relay import Config
+from sentry.models import ProjectKey
+from sentry.web.frontend.base import BaseView
+from sentry.web.helpers import render_to_response
+
+
+SDK_VERSION = '0.5.2'
+DEFAULT_SDK_URL = 'https://s3.amazonaws.com/getsentry-cdn/@sentry/browser/%s/bundle.min.js' % SDK_VERSION
```

Comment:
I don't have an answer for this, but this shouldn't be hardcoded. I think we should make this configurable somehow.